### PR TITLE
Update Dask to version 0.18.1

### PR DESCRIPTION
**Pull request by tomwhite**
**What this PR does / why we need it**:

Updates the Dask chart to the latest stable version of Dask (0.18.1).

**Special notes for your reviewer**:

Tested by running a non-trivial job (scanpy on 1 million rows of zarr data using 5 worker nodes) on a Kubernetes cluster.

### CODE REVIEW

**Comment by tomwhite** [comment]
I've signed the CLA now. /check-cla

**Comment by jakirkham** [comment]
Any news on this PR?

File: stable/dask/README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ The following tables list the configurable parameters of the Dask chart and thei
 | -----------------------      | ---------------------------------| ---------------|
```
Comment (davidkarlsen) [inline stable/dask/README.md:50]: Should  scheduler.imageTag be updated in the readme as well?

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/dask/README.md (hunk 1 above)
Comment (tomwhite) [reply stable/dask/README.md:50]: @davidkarlsen - it's set to `latest` - I thought that would be OK to leave unless I'm missing something.

Comment (davidkarlsen) [reply stable/dask/README.md:50]: But the PR changes the default in values.yaml to tag: "0.18.1" ?

Comment (tomwhite) [reply stable/dask/README.md:50]: Looking again I think you are right. I've updated the default. Thanks @davidkarlsen.

**Pull request merged**